src/app-layer-enip-common.c

```diff
@@ -0,0 +1,810 @@
+/*
+ * Copyright (C) 2014 ANSSI
```

Comment:
Is this correct? I need @inliniac to jump in hear, but this should be 2015, and if the license is BSD then I believe you can keep the copyright, otherwise it should be GPL with the copyright assigned to the OISF (I think).